joomla/joomla-cms
# Show configured upload limit when uploading installer package. (Dependency #11026)

#### Summary of Changes

Displaying max upload limit calculated from PHP configuration values in upload package installer page.

To my knowledge, uploading a file larger than the `post_max_size` limit will still raise PHP warning etc. which is raised even before the Joomla/PHP script is executed. So we do not have any control over it. Therefore, I think it should be better to let the user know the limits beforehand.

Exceeding only the `upload_max_size` (when it is set lower than the `post_max_size`) gives us the chance however to detect a failed upload from our code.

However, since this displays PHP settings to the end user I'd not mind getting this proposal rejected on security grounds. Though, I think only the users having access to installer interface are concerned here.
#### Testing Instructions
- ~~Apply patch https://github.com/joomla/joomla-cms/pull/11026 (dependency) if not merged before you test this one.~~
- Apply this patch.
- Check the installer page - extension package upload tab. It should show the correct limits as configured in your PHP INI. Change the INI settings and make sure the values are reflected correctly again.

![image](https://cloud.githubusercontent.com/assets/6706189/17075964/01b90314-50c3-11e6-9f04-7f0af392bf22.png)
